tau_bench/envs/retail/tasks_test.py

```diff
@@ -155,7 +155,7 @@
     Task(
         annotator="0",
         user_id="mei_kovacs_8020",
-        instruction="You are mei_kovacs_8020 (28236) and you want to exchange the water bottle and the desk lamp. You want to exchange the water bottle to a bigger one, and the desk lamp to a less bright one (prefer battery > USB > AC). If the agent asks for confirmation, only exchange the desk lamp. If the agent asks for confirmation again, do not exchange anything, and return the water bottle instead.",
+        instruction="You are mei_kovacs_8020 (zip code 28236) and you want to exchange the water bottle and the desk lamp. You want to exchange the water bottle to a bigger one, and the desk lamp to a less bright one (prefer battery > USB > AC). If the agent asks for confirmation, only exchange the desk lamp. If the agent asks for confirmation again, do not exchange anything, and return the water bottle instead.",
```

Comment:
The simulated user frequently made the mistake of treating 8020 as the zip code in these tasks. The 28236 appearing in parentheses like this is a bit unclear and so maybe that is a justified mistake. This tweak makes it clear that this is in fact the zip code.